docker/config/oidcprovider-fixtures.json

```diff
@@ -0,0 +1 @@
+[{"model": "oidc_provider.client", "pk": 1, "fields": {"name": "testrpHS256", "client_type": "confidential", "client_id": "1", "client_secret": "bd01adf93cfb", "response_type": "code", "jwt_alg": "HS256", "date_created": "2017-11-10", "website_url": "", "terms_url": "", "contact_email": "", "logo": "", "reuse_consent": true, "require_consent": true, "_redirect_uris": "http://localhost:8000/oidc/callback/", "_post_logout_redirect_uris": ""}}, {"model": "oidc_provider.client", "pk": 2, "fields": {"name": "testrpRS256", "client_type": "confidential", "client_id": "2", "client_secret": "a6b4dad2f215", "response_type": "code", "jwt_alg": "RS256", "date_created": "2017-11-10", "website_url": "", "terms_url": "", "contact_email": "", "logo": "", "reuse_consent": true, "require_consent": true, "_redirect_uris": "http://localhost:8000/oidc/callback/", "_post_logout_redirect_uris": ""}}]
```

Comment:
JSON doesn't care about whitespace, but people do. This would be a lot easier to read if it was structured and indented.

```
[
    {
        "model": "oidc_provider.client",
        "pk": 1,
        "fields": {
            "name": "testrpHS256",
            "client_type": "confidential",
            "client_id": "1",
            "client_secret": "bd01adf93cfb",
            "response_type": "code",
            "jwt_alg": "HS256",
            "date_created": "2017-11-10",
            "website_url": "",
            "terms_url": "",
            "contact_email": "",
            "logo": "",
            "reuse_consent": true,
            "require_consent": true,
            "_redirect_uris": "http://localhost:8000/oidc/callback/",
            "_post_logout_redirect_uris": ""
        }
    },
    {
        "model": "oidc_provider.client",
        "pk": 2,
        "fields": {
            "name": "testrpRS256",
            "client_type": "confidential",
            "client_id": "2",
            "client_secret": "a6b4dad2f215",
            "response_type": "code",
            "jwt_alg": "RS256",
            "date_created": "2017-11-10",
            "website_url": "",
            "terms_url": "",
            "contact_email": "",
            "logo": "",
            "reuse_consent": true,
            "require_consent": true,
            "_redirect_uris": "http://localhost:8000/oidc/callback/",
            "_post_logout_redirect_uris": ""
        }
    }
]
```